### ISSUE TITLE
Live chat module to integrate...

### ISSUE THREAD

**Issue by drosne** [drosne/RMT#44]
I would like to be able to integrate a livechat module. 
I made some searches... it appears that this one is free and working fine... can you please integrate it to wordpress and our website to test it and see if things are matching to our needs. 
https://wordpress.org/plugins/formilla-live-chat/

**Comment by krasizlatev** [comment]
Please go to http://beta.refundmyticket.net/wp-admin/ -> Click Desk -> ClickDesk Webchat to create an account.

**Comment by drosne** [comment]
I have created a login/pass ... it seems that the options are not many in this clickdesk free version. 
why did you choose this one vs the other one ?

**Comment by krasizlatev** [comment]
Please check http://beta.refundmyticket.net/ 
Both are installed in the bottom right corner.

**Comment by drosne** [comment]
well.. i have tested the click desk which appears to be quite powerful... 
the other one is quite nice and ok, however it seems it's not working on mobile.
So i propose to get the formilla out for the moment. 
however i would like to try the one you have used on rentila : 
https://www.olark.com/pricing

**Comment by krasizlatev** [comment]
I would need your account id from 
https://www.olark.com/signup/create_new_account

**Comment by drosne** [comment]
according to what i have tested ... i think that at this step click desk is the best one we tried... 
Please tell me if you have any comment... otherwise you can shut down the other ones.

**Comment by krasizlatev** [comment]
The only thing is that I cannot find out how to make it in French/English depending whether it's
http://beta.refundmyticket.net/
or
http://beta.refundmyticket.net/fr/

Not sure if you want that ?

**Comment by drosne** [comment]
actually ... i didn't too... but i think it's mainly depending on paid or not paid version... 
I don't know too...

----------------------------------------------------------------
Romain DROSNE 
Co-Founder

Claim Assistance 
Get up to 800$ for a delayed flight ! 

www.RefundMyTicket.net
www.RembourserMonBillet.com

@RefundMyTicket
----------------------------------------------------------------

**Comment by krasizlatev** [comment]
Yes, it's quite possible that it's only available in the paid version. Let me know if you want me to send them a support ticket to ask them.

**Comment by drosne** [comment]
nope .. not for now... let's keep that for later on... we will manage this after other priorities which are : 

1- Parsing tool integration (Worldmate)
2- Connecting with EasyVoyage to synchronize things with them
3- SalesForce database Bootstrap
4- Start feeding the Flightdata history

Please give me deadlines about this... 

Romain 


----------------------------------------------------------------
Romain DROSNE 
Co-Founder

Claim Assistance 
Get up to 800$ for a delayed flight ! 

www.RefundMyTicket.net
www.RembourserMonBillet.com

@RefundMyTicket
----------------------------------------------------------------
